src/mvba/abba/message.rs

```diff
@@ -1,6 +1,8 @@
 use blsttc::{Signature, SignatureShare};
 use serde::{Deserialize, Serialize};
 
```

Comment:
Extra line here

Reply:
We generally follow this 3-section import style:
https://github.com/rust-lang/fmt-rfcs/issues/24#issuecomment-252161813



> Three sections, with one blank line after each:
>   -  Standard library imports
>    - External crate imports
>    - Internal imports

Reply:
I always let rustfmt to order the imports. I try to follow it  but this rule is hard to follow.